R/zchunk_L102.GDP.R

```diff
@@ -141,18 +143,19 @@ module_socioeconomics_L102.GDP <- function(command, ...) {
     ## Add region IDs to historical GDP and aggregate by region. Hang onto the country-level data
     ## because we will use them again when we make the IMF adjustments
     gdp_mil90usd_ctry <-
-      left_join_error_no_match(L100.gdp_mil90usd_ctry_Yh, iso_region32_lookup,
-                               by = 'iso') %>%
+      left_join_error_no_match(L100.gdp_mil90usd_ctry_Yh, iso_region32_lookup, by = 'iso') %>%
       rename(gdp = value)
+
     gdp_mil90usd_rgn <- gdp_mil90usd_ctry %>%
+      filter(year %in% HISTORICAL_YEARS) %>%
```

Comment:
I think become the regionally-summed data don't get used again until being matched with the `FUTURE_YEARS` data...I assume we're not responsible, here, for guaranteeing that the user's choices of HY and FY are seamless or make sense.